File: src/AppRegistryService.Contract/Responses/AppInstallerReleaseInfoResponse.cs
namespace AppRegistryService.Contract.Responses;

/// <summary>
/// Defines an application installer info response.
/// </summary>
public sealed class AppInstallerReleaseInfoResponse
{
    /// <summary>
    /// Release info.
    /// </summary>
    public AppReleaseInfo? Release { get; set; }

    /// <summary>
    /// Installer info.
    /// </summary>
    public AppInstallerInfo? Installer { get; set; }
}

File: web/src/responses/AppInstallerReleaseInfoResponse.ts
import { AppInstallerInfo } from "./AppInstallerInfo";
import { AppReleaseInfo } from "./AppReleaseInfo";

/**
 * Defines an application installer info response.
 */
export interface AppInstallerReleaseInfoResponse {
    /**
     * Release info.
     */
    release?: AppReleaseInfo;

    /**
     * Installer info.
     */
    installer?: AppInstallerInfo;
}